cmd/cassandra-reader/main.go

```diff
@@ -77,15 +82,22 @@ func main() {
 }
 
 func loadConfig() config {
+	var logLevel logger.Level
+	err := logLevel.UnmarshalText(mainflux.Env(envLogLevel, defLogLevel))
```

Comment:
Instead of unmarshalling the text in each of the `main`s maybe it would be finally better to just pass the string and unmarshall stuff in `logger.New()` - that can do the validation (by trying to unmarshal text) and return error if it fails.

@anovakovic01 @dusanb94 - opinions?

@mfinley3 sorry about this comment, I know we told you to use lvels struct, but for me it was a bit hard to see without implementation - this patch is pretty big.

Reply:
I liked string initially because of the cleanliness but I do see why passing in an enum for levels makes sense so I'll let @anovakovic01 chime in before implementing anything. I am fine with  going either way on this.

Reply:
@drasko purpose of enumeration is following: if you want to use logger with without environment variable, and set log level explicitly, it's better to use enum constant, than string (because compiler will give you error). In current usage this is not the case, so if you want, you can pass string for now.

Reply:
Cool, converted it back to accepting a string!